fasttree/2.1.11/Dockerfile

```diff
@@ -1,31 +1,54 @@
 # base image
-FROM ubuntu:bionic
+FROM ubuntu:bionic as app
+
+# add ARG here
+ARG fasttree_ver="2.1.11"
 
 # metadata
 LABEL base.image="ubuntu:bionic"
 LABEL version="1"
 LABEL software="Fasttree"
 LABEL software.version="2.1.11"
 LABEL description="Infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences"
-LABEL website="http://www.microbesonline.org/fasttree"
+LABEL website="http://www.microbesonline.org/fasttree/"
 LABEL license="http://www.microbesonline.org/fasttree/"
-LABEL maintainer="Abigail Shockey"
-LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+LABEL dockerfile.version="1"
+LABEL maintainer="Daniel Evans"
+LABEL maintainer.email="evansdr95@gmail.com"
 
 # install ubuntu dependencies
 RUN apt-get update && \
-    apt-get -y install build-essential\
+    apt-get install -y build-essential\
     wget && \
-    apt-get clean
+    apt-get autoclean && rm -rf /var/lib/apt/lists
```

Comment:
We have found that using `--no-install-recommends` reduces the size of the final image. Could you include that? You'll likely need to add `ca-certificates` and `procps` to your apt-get layer.

It would change lines 16-19 to something like

```
RUN apt-get update && apt-get install -y --no-install-recommends \
 wget \
 ca-certificates \
 procps && \
 apt-get autoclean && rm -rf /var/lib/apt/lists/*
 ```

Reply:
Can do!